analytics_data_api/v1/serializers.py

```diff
@@ -569,7 +570,7 @@ def get_enrollment_modes(self, obj):
         return obj.get('enrollment_modes', None)
 
     def get_programs(self, obj):
-        return obj.get('programs', None)
+        return list(obj.get('programs', set()))
```

Comment:
Similar to how if `d = {'x': 10}`, you might write `str(d.get('x'), 0)` instead of `str(d.get('x'), '0')`.